.changeset/emotion-update-new.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma": minor
```

Comment:
This is technically correct because we are updating all the packages, but I think this will fail the automation as it expects one of the package names. I think we actually will need to make a changeset file for each of the packages so that when we release, all of the packages get updated:
- react-magma-dom
- @react-magma/charts
- @react-magma/dropzone
- @cengage-patterns/header
(we can skip docs)

Reply:
Done 👍